models/db/index.go

```diff
@@ -55,8 +58,25 @@ func SyncMaxResourceIndex(ctx context.Context, tableName string, groupID, maxInd
 	return nil
 }
 
+func postGresGetNextResourceIndex(ctx context.Context, tableName string, groupID int64) (int64, error) {
```

Comment:
```suggestion
func postgresGetNextResourceIndex(ctx context.Context, tableName string, groupID int64) (int64, error) {
```
capitalizing like that seems odd.